include/AtlasTypes.hpp

```diff
@@ -20,6 +20,8 @@
 #define PT_ENTRIES_SV32 1024 // Sv32 has 1,024 entries in a page table
 #define PT_ENTRIES_SV39 512  // Sv39 has 512 entries in a page table
 
+#define PTE_SIZE 4
```

Comment:
I don't think this needs to be here. We should be able to just check for RV32 or RV64 to get the PTE size.